pimd/pim_cmd.c

```diff
@@ -1326,79 +1329,94 @@ static void igmp_show_statistics(struct pim_instance *pim, struct vty *vty,
 			igmp_stats_add(&igmp_stats, &igmp->igmp_stats);
 		}
 	}
+
```

Comment:
Why not just put the `if (!found_ifname) .... ` logic here and we don't need to rewrite large chunks of this awful code

Reply:
Fixed.